pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -3,970 +3,970 @@ source: pdg/src/main.rs
-	n[57]: addr.load  n[56] => _   @ bb12[2]:  fn insertion_sort; _20 = (*_21);                                        
-	n[58]: copy       n[6]  => _31 @ bb13[4]:  fn insertion_sort; _31 = _2;                                            
-	n[59]: offset[0]  n[58] => _30 @ bb15[5]:  fn insertion_sort; _30 = offset(move _31, move _32);                    
-	n[60]: addr.load  n[59] => _   @ bb16[2]:  fn insertion_sort; _29 = (*_30);                                        
-	n[61]: copy       n[6]  => _38 @ bb16[6]:  fn insertion_sort; _38 = _2;                                            
-	n[62]: offset[1]  n[61] => _37 @ bb16[11]: fn insertion_sort; _37 = offset(move _38, move _39);                    
-	n[63]: addr.store n[62] => _   @ bb17[2]:  fn insertion_sort; (*_37) = move _29;                                   
-	n[64]: copy       n[6]  => _47 @ bb24[8]:  fn insertion_sort; _47 = _2;                                            
-	n[65]: offset[0]  n[64] => _46 @ bb24[13]: fn insertion_sort; _46 = offset(move _47, move _48);                    
-	n[66]: addr.store n[65] => _   @ bb25[2]:  fn insertion_sort; (*_46) = move _45;                                   
-	n[67]: copy       n[6]  => _10 @ bb3[4]:   fn insertion_sort; _10 = _2;                                            
-	n[68]: offset[4]  n[67] => _9  @ bb3[9]:   fn insertion_sort; _9 = offset(move _10, move _11);                     
-	n[69]: addr.load  n[68] => _   @ bb5[2]:   fn insertion_sort; _8 = (*_9);                                          
-	n[70]: copy       n[6]  => _22 @ bb9[5]:   fn insertion_sort; _22 = _2;                                            
-	n[71]: offset[3]  n[70] => _21 @ bb11[5]:  fn insertion_sort; _21 = offset(move _22, move _23);                    
-	n[72]: addr.load  n[71] => _   @ bb12[2]:  fn insertion_sort; _20 = (*_21);                                        
-	n[73]: copy       n[6]  => _47 @ bb24[8]:  fn insertion_sort; _47 = _2;                                            
-	n[74]: offset[4]  n[73] => _46 @ bb24[13]: fn insertion_sort; _46 = offset(move _47, move _48);                    
-	n[75]: addr.store n[74] => _   @ bb25[2]:  fn insertion_sort; (*_46) = move _45;                                   
+	n[0]:  &_35       _     => _34 @ bb30[4]:  fn main_0;         (mut n[75], load n[72]) _34 = &mut _35                                      
+	n[1]:  copy       n[0]  => _40 @ bb30[12]: fn main_0;         ()                      _40 = &(*_34)                                       
+	n[2]:  copy       n[1]  => _39 @ bb30[13]: fn main_0;         ()                      _39 = move _40 as &[i32] (Pointer(Unsize))          
+	n[3]:  copy       n[2]  => _1  @ bb0[0]:   fn len;            ()                      _38 = len(move _39)                                 
+	n[4]:  copy       n[0]  => _42 @ bb31[6]:  fn main_0;         (mut n[75], load n[72]) _42 = &raw mut (*_34)                               
```

Comment:
i feel like the surrounding `n[_]` might be a little too verbose and redundant -- the node id's should be enough

Reply:
Sounds good.

Reply:
If we change that, I think we should change it everywhere we use `n[_]`.